vm-setup-script-usage.md

```diff
@@ -0,0 +1,92 @@
+# Orka VM Setup Script Usage
+## This script automates the configuration and setup of macOS Orka VMs for optimal performance. It configures system settings, installs required tools, and prepares the VM for use with Orka.
+
+### System requirements:
+
+- macOS IPSW file (Tahoe, Sequoia, or Sonoma)
+- Admin/sudo privileges on the VM
+- Internet connection for downloading packages
+- Terminal access (via SSH, Screen Sharing, or direct console)
+
+### Before running:
+
+- Ensure you have administrator access to the VM
+- Close any important applications (they will be terminated during cleanup)
+- Save any work in progress (the VM will reboot automatically)
+
+### Installation:
+
+#### Download via GitHub
+
+- ```/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/macstadium/orka-images/orka-ipsw-setup.sh)"```
```

Comment:
You do not need all 3 lines here. Again - see the Homebrew example I've pointed out several times now :)) 

This single line will download the script with curl and execute it directly with bash.

Reply:
updated this, thanks

Reply:
Also - this does not render correctly.

When we use triple backticks, the correct way is to add the language hint directly after the opening backticks, and then separate on newlines. Like this:

````
```bash
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/macstadium/orka-images/orka-ipsw-setup.sh)"
```
````

Which renders as this:

```bash
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/macstadium/orka-images/orka-ipsw-setup.sh)"
```

<img width="991" height="185" alt="Screenshot 2025-10-02 at 11 28 32 AM" src="https://github.com/user-attachments/assets/2a3fdf60-9c63-4c76-853e-31888b04afe5" />